examples/nlp/language_modeling/conf/megatron_bert_config.yaml

```diff
@@ -9,7 +9,7 @@ trainer:
   logger: False # logger provided by exp_manager
   enable_checkpointing: False
   replace_sampler_ddp: False
-  max_epochs: null
+  max_epochs: 1000 # PTL default. In practice we don't usually train for more than 1 epoch.
```

Comment:
Doesn't this fail cause PTL expects one or the other, not both ?

Reply:
No it will end training w/e the first is reached.